dapps/marketplace/src/components/OgnBadge.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import { fbt } from 'fbt-runtime'
+
+class OgnBadge extends Component {
```

Comment:
Since there's no state we can switch to a function component?

```
const OgnBadge = ({ amount, className }) =>
  <span...
```

Reply:
good point thanks 👍